src/main/java/org/digma/intellij/plugin/toolwindow/DigmaLeftToolWindowListener.java

```diff
@@ -0,0 +1,17 @@
+package org.digma.intellij.plugin.toolwindow;
+
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.openapi.wm.ex.ToolWindowManagerListener;
+import org.digma.intellij.plugin.PluginId;
+import org.digma.intellij.plugin.posthog.ActivityMonitor;
+import org.jetbrains.annotations.NotNull;
+
+public class DigmaLeftToolWindowListener implements ToolWindowManagerListener {
```

Comment:
please change the name of this listener to keep our naming consistent as we don't have any LeftTool window.
we have **`DigmaSidePaneToolWindowFactory`** (now by default it's being displayed on right side by Rony's request, so please avoid hardcoded names like "LeftToolWindow")